tomcat/auto_conf.yaml

```diff
@@ -0,0 +1,10 @@
+ad_identifiers:
+  - tomcat
+
+init_config:
+  - is_jmx: true
+    collect_default_metrics: true
+
+instances:
+  - url: "%%host%%"
+    port: "9012"
```

Comment:
where does this port come from? It's not listed in https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers - is it considered a de fact standard? Does it work in 90%+ of tomcat setups?

Reply:
The port is also used in the regular `tomcat.yaml` configuration. But I think the `9012` is the default port tomcat uses for monitoring (https://tomcat.apache.org/tomcat-7.0-doc/monitoring.html)